pkg/execution/driver/request.go

```diff
@@ -41,6 +41,9 @@ type SDKRequestContext struct {
 	// Attempt is the zero-index attempt number.
 	Attempt int `json:"attempt"`
 
+	// MaxAttempts is the maximum number of attempts allowed for this function.
+	MaxAttempts int `json:"max_attempts"`
```

Comment:
Makes sense and is solid.  Alternatively, you could fetch the latest config from the SDK itself.